Source/IntegrationTests/IntegrationTests.csproj

```diff
@@ -41,6 +41,12 @@
         <Content Include="..\..\node_modules\**\*.*" LinkBase="TestFiles\LitTests\LitTest\node_modules">
             <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
         </Content>
+        <Content Update="..\..\Test\linters\constructorCaseWithoutParenthesis.dfy">
```

Comment:
Why do you need these lines? I haven't seen someone modify IntegrationTests.csproj in a while